.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,116 @@
+name: Publish Release
+on: workflow_dispatch
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Checkout code
+      uses: actions/checkout@v2
+      with:
+        submodules: recursive
+        ref: ${{ github.event.pull_request.head.sha }}
+
+    - name: Setup node version
+      uses: actions/setup-node@v2
+      with:
+        node-version: '14'
+        cache: 'npm'
+        cache-dependency-path: '**/package-lock.json'
+
+    - name: Setup NPM version
+      run: npm install -g npm@7
+
+    - name: Get NPM cache directory
+      id: npm-cache-dir
+      run: echo "::set-output name=dir::$(npm config get cache)"
+
+    - name: Restore NPM cache
+      id: npm-cache
+      uses: actions/cache@v2
+      with:
+        path: ${{ steps.npm-cache-dir.outputs.dir }}
+        key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-node-
+
+    - name: Install node modules
+      run: npm ci --ignore-scripts
+
+    - name: Download prebuild PR artifacts
```

Comment:
Will this also obtain the `.xcframework` and Android build artifacts?

Reply:
once all builds in the PR workflow are enabled then it should contain them.

Reply:
Ok, just wanted to make sure.  The example you linked only contains the `node` binaries